### DISCUSSION TITLE
What are you using Agate for?

### DISCUSSION THREAD

**Discussion by mbrubeck** [mbrubeck/agate#11] [Show and tell]
I'd love to hear what people are doing with Agate.  If you have an Agate-powered Gemini server we can visit, please feel free to leave a link in this discussion!

**Comment by steko** [comment]
Hello, thanks for creating Agate!

I launched gemini://gemini.iosa.it a few days ago and it was great to setup a working server so quickly.

**Comment by leematthewjackson** [comment]
For my own basic static site containing personal thoughts and links to my channels etc gemini://leematthewjackson.space

Super easy to configure thank you!

**Reply by Johann150** [reply]
Did you by chance use the install script? (`tools/debian/install.sh`)

**Comment by laosb** [comment]
Using agate to serve my personal Gemini site at gemini://gmi.of.sb . So far so good!

I do wish some basic CGI capabilities so that I can write a Bash script or some basic programs to do some simple interactivities.

**Reply by Johann150** [reply]
unfortunately CGI is not something we will implement in Agate, for more information please see #35

**Comment by jbanana** [comment]
I set up Agate for a personal capsule at gemini://freeshell.de/
Setup was pretty easy: create a couple of dirs, get a binary and run.

**Reply by ghost** [reply]
Jbanana are you running freeshell too? It’s a cool service.

**Reply by jbanana** [reply]
No, freeshell.de is nothing to do with me except I have an account there.

**Comment by ghost** [comment]
I’m running cugi.ie from it and I know eir.mooo.com runs agate too.
